catalog/imaging/catalog-imaging-transformer-chip/src/main/java/com/connexta/alliance/imaging/chip/transformer/ImagingRouteBuilder.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) Connexta, LLC
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package com.connexta.alliance.imaging.chip.transformer;
+
+import org.apache.camel.builder.RouteBuilder;
+
+/**
+ * a Camel RouteBuilder to define the route from and to the MetacardTransformer proxy object.
+ */
+public class ImagingRouteBuilder extends RouteBuilder {
```

Comment:
I could, but this is consistent with org.codice.ddf.catalog.content.monitor.ContentDirectoryMonitor.